redaxo/src/addons/backup/lib/cronjob.php

```diff
@@ -7,6 +7,9 @@ class rex_cronjob_export extends rex_cronjob
 {
     public const DEFAULT_FILENAME = '%REX_SERVER_%Y%m%d_%H%M_rex%REX_VERSION';
 
+    /**
+     * @return bool
+     */
     public function execute()
```

Comment:
@staabm Brauchen wir es wirklich in Kindklassen auch?
Meines Wissens ist das nicht notwendig, wenn der Typ nicht abweicht im Vergleich zur Elternklasse.

Reply:
Nachher hatte ich es weggelassen. Es kam auf die Reihenfolge der Bearbeitung an. Wenn die Elternelemente definiert waren, wurden die Kindklassen nicht mehr aufgeführt. Ich hatte es zu spät gemerkt. Ich kann es nochmals durchschauen und bearbeiten.

Reply:
in elternklassen sollte ausreichen